src/modules/loki/test/ReadoutGenerator.cpp

```diff
@@ -23,11 +23,11 @@ uint16_t lokiReadoutDataGen(uint16_t DataSections, uint16_t DataElements, uint8_
   auto DP = (uint8_t *)Buffer;
   //printf("Buffer pointer %p\n", (void *)Buffer);
   auto Header = (ReadoutParser::PacketHeaderV0 *)DP;
-  #ifdef READOUT_EXTRA_PADDING
-  Header->Padding = 0;
-  #endif
-  Header->CookieVersion = 0x00535345;
-  Header->TypeSubType = 0x30;
+    Header->CookieAndType = 0x30535345;
```

Comment:
The ownCloud document has 0x52 for `S` (page 3).

Reply:
Good thing we're not reviweing that then ;-) 0x53 is the correct value.
Afonso, you're not supposed to be this thorough when reviewing, but then 
again you didn't find the typo in the document name 'readut'  :-)
But thanks for noticing, I will make a correction

Reply:
I did miss that 😄